frontend/src/api/omni/specs/omni.pb.ts

```diff
@@ -699,6 +699,7 @@ export type UserPilotSettings = {
 
 export type StripeSettings = {
   enabled?: boolean
+  min_commit?: number
```

Comment:
We definitely need to add dirty check. Do we need to include this on 1.4 release?